consul/base.py

```diff
@@ -1928,146 +1928,117 @@ def list(self, token=None):
-                payload["Type"] = token_type
-            if rules:
-                assert isinstance(rules, str), "Only HCL or JSON encoded strings supported for the moment"
-                payload["Rules"] = rules
-            if acl_id:
-                payload["ID"] = acl_id
-
-            data = json.dumps(payload) if payload else ""
-
-            return self.agent.http.put(CB.json(is_id=True), "/v1/acl/create", params=params, data=data)
-
-        def update(self, acl_id, name=None, token_type=None, rules=None, token=None):
+        def clone(self, accessor_id, token=None, description=""):
             """
-            Updates the ACL token *acl_id*. This is a privileged endpoint, and
-            requires a management token. *token* will override this client's
-            default token. An *ACLPermissionDenied* exception will be raised if
-            a management token is not used.
-
-            *name* is an optional name for this token.
-
-            *token_type* is either 'management' or 'client'. A management token is
-            effectively like a root user, and has the ability to perform any
-            action including creating, modifying, and deleting ACLs. A client
-            token can only perform actions as permitted by *rules*.
-
-            *rules* is an optional `HCL`_ string for this `ACL Token`_ Rule
-            Specification.
-
-            Returns the string *acl_id* of this token on success.
+            Clones the token identified by *accessor_id*. This is a privileged endpoint, and requires a token with acl:write.
+            :param accessor_id: The accessor ID of the token to clone
+            :param token: token with acl:write capability
+            :param description: Optional new token description
+            :return: The cloned token information
             """
             params = []
             token = token or self.agent.token
             if token:
                 params.append(("token", token))
 
-            payload = {"ID": acl_id}
-            if name:
-                payload["Name"] = name
-            if token_type:
-                assert token_type in ("client", "management"), "token_type must be client or management"
-                payload["Type"] = token_type
-            if rules:
-                assert isinstance(rules, str), "Only HCL or JSON encoded strings supported for the moment"
-                payload["Rules"] = rules
-
-            data = json.dumps(payload)
-
-            return self.agent.http.put(CB.json(is_id=True), "/v1/acl/update", params=params, data=data)
+            json_data = {"Description": description}
+            return self.agent.http.put(
+                CB.json(),
+                f"/v1/acl/token/{accessor_id}/clone",
+                params=params,
+                data=json.dumps(json_data),
+            )
 
-        def clone(self, acl_id, token=None):
+        def create(self, token=None, accessor_id=None, secret_id=None, description=""):
```

Comment:
No more token type?

Reply:
wdym ?

Reply:
token type doesn't exist anymore. It was relevant in prior consul version.
A "management" token is just a token with acl:write capability now